### PR TITLE
Added 127505 Tanks

### PR DESCRIPTION
**Pull request by sbender9**
Added 127505 Tanks

### CODE REVIEW

File: n2kMappings.js [hunk 1]
```diff
@@ -392,8 +392,29 @@ exports.mappings = {
...
     },
     source: 'Actual Temperature'
-  }]
+    }],
 
+  //Tanks
+  '127505': [{
+      source: 'Level',
+      node: function(n2k) { 
+	return 'tanks.' + n2k.fields['Type'] + '_' + n2k.fields['Instance'] + '.currentLevel;'
```
Comment (tkurki) [inline n2kMappings.js]: Extra semicolon here - the test fails?

Comment (sbender9) [inline n2kMappings.js]: Fixed that, but it still doesn't like it. Also changed to just be 'fuel' in the path because it was complaining.

**Comment by sbender9** [comment]
Getting closer here. I can't figure out how to set capacity without getting the source, pgn, etc. in there.

**Comment by tkurki** [comment]
It is not you, it is the schema and the way n2k-signalk works. Capacity is not defined as a `numberValue` in the schema, meaning that it does not have the `source` information or the value/values construct (for handling multiple sources for the same value) like for example `currentLevel` has. 

But currentLevel is a bit off too, it should be returned as a single value, not `valueNumber`. 

Why don't we leave this here for now, I'll bang this together and probably make the necessary schema change on the side.

**Comment by sbender9** [comment]
Similar issues with batteries. 

I'm I looking at an old schema?:

```
            "currentLevel": {
              "description": "Level of fluid in tank 0-100%",
              "$ref": "../definitions.json#/definitions/numberValue",
              "units": "ratio"
            },
```

**Comment by sumps** [comment]
You will find a lot of examples in the Spec like this, where it was originally defined as "What data we would like to " and then later on the "What data we can actually measure". In most cases we have kept the "Like to Have" data field as there may be some clever person out there that designs a way to get that data. 

Also it is not unreasonable to expect a server to receive the "Level" value from the N2K network 0% to 100% and then if the boat builder or owner has entered the tank capacity in a data field, the server can derive the number of Litres or Gallons remaining and transmit this in another data field..

**Comment by tkurki** [comment]
While what Paul says is true it is not the issue that you are facing. And the problem is with `capacity`, not with `currentLevel`.

The logic that converts the delta to full in n2k-signalk and SK node server assumes that the value in delta is either an object like `{latitude:..., longitude:...}` (as in navigation.position) or a simple value like a number. It will attach source, pgn etc to either.

Signal K `numberValue` with value/values/source etc is defined like https://github.com/SignalK/specification/blob/daaddf627fbe116ec2be478a18a7212307c15acd/schemas/groups/tanks.json#L37-L41

and capacity's type is defined as  number https://github.com/SignalK/specification/blob/daaddf627fbe116ec2be478a18a7212307c15acd/schemas/groups/tanks.json#L33

Btw #9  is the same pgn.

**Comment by tkurki** [comment]
Replaced by #32

**Pull request closed without merging**